### PR TITLE
Create dist files for browser support (fixes #12)

### LINKED ISSUE #12: Add a browserified version to the repo
Whenever we do a release, we should create `./dist/leche.js` and publish it to the repo and npm. This would allow developers to access Leche in two new ways:

1. Using RawGit to access the file directly
2. Linking to the browserified version if they have the npm package installed

### PR DESCRIPTION
**Pull request by nzakas**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @nzakas has signed the CLA. Thanks for the pull request!

File: dist/leche.js [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+			}
+		}
+	}
+
+};
+
+},{"mocha":2}],2:[function(require,module,exports){
+
+},{}]},{},[1])(1)
+});
```
Comment (j3tan) [inline dist/leche.js]: nitpick: I don't suppose we can insert that new line

**Comment by j3tan** [comment]
+2

File: dist/leche.js (hunk 1 above)
Comment (nzakas) [inline dist/leche.js]: Sure thing.

**Pull request merged**